src/components/Session.astro

```diff
@@ -17,6 +18,17 @@ const {
 	talk: { title, people, tags, description, slug },
 	...rest
 } = Astro.props;
+
+const schedule = days
+	.flatMap((day) =>
+		day.activities.flatMap((activity) =>
+			activity.within
+				?.filter((inner) => inner.type === "session" && inner.session === slug)
+				.map((activity) => ({ activity, day })),
+		),
+	)
+	.filter((x) => !!x?.activity)
+	.at(0)!;
```

Comment:
I don't love this code, but it's a static website for a conference. We'll live with it. 😄